docs/source/cloud_integration.md

```diff
@@ -96,5 +96,5 @@ To run NVTabular on the cloud using GCP, do the following:
 
 5. Run the container using the following command:
    ```
-   docker run --gpus all --rm -it -p 8888:8888 -p 8797:8787 -p 8796:8786 --ipc=host --cap-add SYS_PTRACE -v /mnt/raid:/raid nvcr.io/nvidia/nvtabular:0.3 /bin/bash
+   docker run --gpus all --rm -it -p 8888:8888 -p 8797:8787 -p 8796:8786 --ipc=host --cap-add SYS_PTRACE -v /mnt/raid:/raid nvcr.io/nvidia/nvtabular:0.4 /bin/bash
```

Comment:
This isn't the containers we're going with - we should either leave on the 0.3 tag for now (which is valid) or switch over to the merlin containers for 0.4  (https://docs.google.com/document/d/1fJqK-NZnY7xIUGoPhXRMPjmwSRz--VNR0z9pb0_fptU/edit?disco=AAAALZ0SuYI&ts=60394597&usp_dm=false) . 

I think for now lets just leave at 0.3 - we will have other updates/version bumps needed for release

Reply:
I just added 0.3 back.